include/tdl/ParamValue.h

```diff
@@ -931,10 +890,12 @@ namespace tdl
     return *(data_.str_);
   }
 
+  //!\cond
   ParamValue::operator std::vector<std::string>() const
   {
     return this->toStringVector();
   }
+  //!\endcond
```

Comment:
why is this one excluded?

Reply:
For some reason it gets documented as global. If I exclude this, the docs are fine (including the member).